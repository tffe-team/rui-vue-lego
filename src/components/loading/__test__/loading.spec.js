import $loading from '../index'
import {later} from '../../../test'

test('create a loading', async () => {
  $loading.show()
  await later()
  expect($loading.$el.outerHTML).toMatchSnapshot()
})
