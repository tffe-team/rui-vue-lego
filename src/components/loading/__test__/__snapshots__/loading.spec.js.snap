// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create a loading 1`] = `"<div class=\\"r-vue-lego-loading vue-lego-loading vue-lego-loading-common\\" style=\\"z-index: 10000;\\" id=\\"$loading-common-el\\"><div class=\\"inner\\"><i class=\\"rui-ico\\"></i> <p class=\\"tips\\"></p></div></div>"`;
